tests/spec/w3c/style-spec.js

```diff
@@ -172,15 +181,25 @@ describe("W3C - Style", () => {
     expect(elem.content).toBe(expectedStr);
   });
 
-  it("should default to base when specStatus is missing", async () => {
-    await loadWithStatus("", "https://www.w3.org/StyleSheets/TR/2021/base.css");
-  });
-
-  specStatus.forEach(test => {
-    it(`should style according to spec status ${test.status}`, async () => {
-      await loadWithStatus(test.status, test.expectedURL);
-    });
-  });
+  it(
+    "styles according to specStatus and group",
+    async () => {
+      for (const { specStatus, expectedURL, group } of statuses) {
+        const conf = {
+          specStatus,
+          group,
+        };
+        const ops = makeStandardOps(conf);
+        const doc = await makeRSDoc(ops);
+        const query = `link[href^='${expectedURL}']`;
+        const elem = doc.querySelector(query);
+        const context = JSON.stringify(conf);
+        expect(elem).withContext(context).toBeTruthy();
+        expect(elem.href).withContext(context).toBe(expectedURL);
+      }
+    },
+    jasmine.DEFAULT_TIMEOUT_INTERVAL * 3
+  );
```

Comment:
Not sure, but maybe we can switch for-loop order like:
```js
for (const { specStatus, expectedURL, group } of statuses) {
  it(`styles with specStatus: ${specStatus}; group: ${group}`, async() => {
    // makeRsDoc();
    // expect() ...
    // ...
  });
}
```

Should have cleaner assertion failures and not require custom timeouts then. If above works, might make sense to collect them together in a `describe()` block.

Reply:
Ironically, that's what it was originally 😬... I'm 50/50 on this one... without wasting too much time one it, maybe I could get it to produce all the documents are once, then `Promise.all()` the documents, and do the checks that way. 

That, in theory should be faster... and `.context()` can tell us which one failed.

Reply:
😬 didn't see what was before. I thought you just added timeout

Reply:
Creating the docs all at once ended up making the test twice as long, so reverted to your solution 👍.